### DISCUSSION TITLE
Can we export and re-import all the data

### DISCUSSION THREAD

**Discussion by danielnetop** [firefly-iii/firefly-iii#5089] [Firefly III] [answered]
Is it possible to run some kind of export and then re import all the information back into the install of Firefly?

One scenario is to test locally for example using sqlite (in order just to test it without having to setup databases) and then easily export from one install into another
Another can be merging two installations, we have firefly.domain1.tld and another install at firefly.domain2.tld and each one has different user, can we export from domain2 and import it on domain1 so that the user can register an account on domain1 and save costs and not losing any data?

**Comment by JC5** [comment, marked as answer]
There is no such feature I'm afraid.

**Reply by danielnetop** [reply]
Got it, thank you 👍
